cmd/template_cmd.go

```diff
@@ -447,13 +455,12 @@ func convertFlagParamsToConfigurationAttributesFromFile(c *cli.Context, casIn st
 			buff += line
 		}
 		if err := scanner.Err(); err != nil {
-			return nil, fmt.Errorf("reading standard input: %s", err)
+			return nil, fmt.Errorf("cannot read from standard input: %v", err)
 		}
 
-		if err := json.Unmarshal([]byte(buff), &content); err != nil {
+		if err := json.NewDecoder(strings.NewReader(buff)).Decode(&content); err != nil {
```

Comment:
Why don't you use `json.NewDecoder(os.Stdin)`? That way you wouldn't need
- to store in memory the whole STDIN content
- to manually scan all the lines in os.Stdin
- to have json.NewDecoder inside the `if casIn == "-" `

Reply:
done, it simplifies and works great!